### PR TITLE
rd

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,284 @@
...
+```js
+  return `Hello, student! It's a fine day to learn JavaScript`
+//=> "Hello, student! It's a fine day to learn JavaScript"
+```
+
+## Define the Term "Closure"
+
+In the previous example, we could call the "inner" function, the **anonymous
+function** a "closure." It "encloses" the function-level scope of its parent.
+And, like a backpack, it can carry out the knowledge that it saw - _even when
```
Comment (maxwellbenton) [inline README.md]: I like backpacks

File: README.md [hunk 2]
```diff
@@ -1,16 +1,284 @@
...
+```
 
 ## Conclusion
 
+In this lesson, we've covered the basics of function declaration, invocation,
+and function scope. As a refresher on your skills, we've provided a simple lab
+to make sure that you're set for the new information coming up in the rest of
+this module.
+
 ## Resources
```
Comment (maxwellbenton) [inline README.md]: remove

File: README.md [hunk 3]
```diff
@@ -1,16 +1,284 @@
...
+- Define the term "anonymous function"
+- Define an IIFE: Instantly-Invoked Function Expression
+- Define the term "function-level scope"
+- Define the term "closure"
 
 ## Introduction
 
-## SWBAT 1
+This lab provides a quick summation of the basics of JavaScript functions. You
+should have encountered all these ideas before now, but we're going to give a
```
[the remark this answers is not in the document]
Comment (sgharms) [reply README.md]: Hopefully, they will be because they did our 3.1 ;) But I've removed this language.

File: README.md (hunk 1 above)
Comment (sgharms) [reply README.md]: 🔙 👝 just for you.

**Pull request merged**